From 53cfe9f18c1414d211690a765aecfff3bbe6ea1b Mon Sep 17 00:00:00 2001
From: Suren Baghdasaryan <surenb@google.com>
Date: Thu, 20 Feb 2020 17:38:36 -0800
Subject: [PATCH] defconfig: Disable CONFIG_RT_GROUP_SCHED kernel config option

Android does not use per-cgroup CPU allowance for RT tasks feature.
Disable it so that RT task allowance is controlled globally using
/proc/sys/kernel/sched_rt_runtime_us.

Bug: 149954332
Test: TreeHugger
Change-Id: I63fe994732d9abb0119a79870f4c3c772db75aa7
Suggested-by: Snild Dolkow <snild@sony.com>
Signed-off-by: Suren Baghdasaryan <surenb@google.com>
Merged-In: I63fe994732d9abb0119a79870f4c3c772db75aa7
---


diff --git a/arch/arm/configs/elementalx_defconfig b/arch/arm/configs/elementalx_defconfig
index 731b872..47f66c0 100644
--- a/arch/arm/configs/elementalx_defconfig
+++ b/arch/arm/configs/elementalx_defconfig
@@ -97,7 +97,7 @@ CONFIG_CGROUP_PERF is not set
 CONFIG_CGROUP_SCHED=y
 CONFIG_FAIR_GROUP_SCHED=y
 # CONFIG_CFS_BANDWIDTH is not set
-CONFIG_RT_GROUP_SCHED=y
+# CONFIG_RT_GROUP_SCHED is not set
 CONFIG_BLK_CGROUP=y
 # CONFIG_CHECKPOINT_RESTORE is not set
 CONFIG_NAMESPACES=y
-- 
2.7.4

